apps/server/src/modules/files/repo/sync-files.repo.ts

```diff
@@ -1,15 +1,55 @@
 import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
 import { Injectable } from '@nestjs/common';
-import { EntityId, FileRecordParentType, Task } from '@shared/domain';
+import { PaginationParams } from '@shared/controller';
+import { Counted, EntityId, FileRecordParentType, Task } from '@shared/domain';
 import { SyncFileItemMapper } from '../mapper';
 import { SyncFileItem, SyncFileItemData } from '../types';
 
+type Parent = {
+	_id: EntityId;
+	schoolId: EntityId;
+	filesId: EntityId;
+};
+
 // This repo is used for syncing the new filerecords collection with the old files collection.
 // It can be removed after transitioning file-handling to the new files-storage-microservice is completed.
 @Injectable()
 export class SyncFilesRepo {
 	constructor(protected readonly _em: EntityManager) {}
 
+	async loadParent(pagination: PaginationParams, timestamp: Date): Promise<Parent[]> {
+		const result = await this._em.aggregate(Task, [
+			{
+				$match: { fileIds: { $exists: true, $ne: [] } },
+			},
+			{
+				$expr: {
+					$gt: ['updatedAt', timestamp],
```

Comment:
Can we use the updatedAt of the parent? Is the parent always updated when a file is updated?